### ISSUE TITLE
Migrate and connect module account and authenticate

### ISSUE THREAD

**Issue by lpirola** [nossas/bonde#361]

**Comment by gabrielrtakeda** [comment]
@igr-santos, about the migration of accounts module routes, I might use this issue in parallel with you or, it is better to create another issue only for routes migration?

**Comment by igr-santos** [comment]
Closes #429
